docs/workshops.md

```diff
@@ -26,26 +26,27 @@ We have standard training tutorial modules on GNPS. Check it out here:
 ## Previous Workshops
 
 | Venue        | Description          | Dates | Material | Estimated Attendance |
-| ------------- |-------------| -----| -----| ----- | 
+| ------------- |-------------| -----| -----| ----- |
```

Comment:
I think this is already listed above, can we just move what we have up there and move it down?